lpcr_pos/static/src/xml/Screens/ProductScreen/ActionpadWidget.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!-- Copyright 2025 CoopITEasy - Simon Hick <sim@coopiteasy.be>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html) -->
+<templates xml:space="preserve">
+    <t
+        t-name="pos_hide_refund_button.ActionpadWidgetHideRefundButton"
+        t-inherit="point_of_sale.ActionpadWidget"
+        t-inherit-mode="extension"
+    >
+        <xpath
+            expr="//button[hasclass('button','pay','validation')]"
+            position="attributes"
+        >
+            <attribute name="t-attf-class">
+            button pay validation {{ (props.actionToTrigger == 'do-refund' and env.pos.user.role != 'manager') ? 'hide-mode' : ''}}
+            </attribute>
+        </xpath>
+    </t>
```

Comment:
2 remarks:

1.  to add a class conditionally, the [`t-att-class` attribute](https://github.com/odoo/owl/blob/master/doc/reference/templates.md#dynamic-class-attribute) is cleaner: it allows to add a class without needing to copy the existing classes.
2.  why not use the `t-if` attribute with the same conditions (negated) instead of using a class? does it break something?